### PR TITLE
chore(#691): seed development environment manually

### LINKED ISSUE #691: Update demo to use seed data instead of profiles.yml
Currently these deployments use a `lightdash.yml` to setup an initial project programatically (i.e. without needing the UI). They should be updated to use seed data into the database since we're dropping support for specifying projects in `lightdash.yml` files. 

- [ ] Demo
- [ ] Dev environment
- [ ] Pull request preview environment

### PR DESCRIPTION
**Pull request by owlas**
We seed an initial user and project, including warehouse credentials, for development. Previously this was done using `lightdash.yml`

Also added a utility to seed the demo.lightdash.com deployment.

Closes #691

—

Created via [Raycast](https://www.raycast.com?ref=signatureGithub)

### CODE REVIEW

File: packages/backend/src/database/seeds/development/01_initial_user.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    SEED_ORGANIZATION,
+    SEED_ORGANIZATION_MEMBERSHIP,
+    SEED_PASSWORD,
+    SEED_USER,
+} from 'common';
+
+export async function seed(knex: Knex): Promise<void> {
+    // Deletes ALL existing entries
+    await knex('users').del();
+    await knex('organizations').del();
```
Comment (ZeRego) [inline packages/backend/src/database/seeds/development/01_initial_user.ts:14]: Should we also clear emails, organization_memberships and password_logins tables ? 
If so, we should also clear all the tables in the other seed scripts.

Comment (owlas) [reply packages/backend/src/database/seeds/development/01_initial_user.ts:14]: These get dropped with `CASCADE` after the user. Would you prefer it was manual?

File: lightdash.yml [hunk 2]
```diff
@@ -1,7 +1,3 @@
 version: '1.0'
 
-projects:
-  - name: default
-    type: 'dbt'
-    profiles_dir: /usr/app/profiles
-    project_dir: /usr/app/dbt
```
Comment (ZeRego) [inline lightdash.yml:7]: We need to keep this since projects is required. There is a ticket to fix it: https://github.com/lightdash/lightdash/issues/520

Comment (owlas) [reply lightdash.yml:7]: Whoops! Didn't mean to include that, thanks!

**Review by ZeRego — APPROVED** [review]

**Pull request merged**